compiler/test/fail_compilation/fail15414.d

```diff
@@ -0,0 +1,20 @@
+// https://issues.dlang.org/show_bug.cgi?id=15414
+
+// REQUIRED_ARGS: -de
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail15414.d(19): Deprecation: `__traits(getAttributes)` may only be used for individual functions, not overload sets such as: `fun`
```

Comment:
perhaps add a supplemental error to suggest using `__traits(getOverloads,...)` to select the desired function to extract attributes from?